src/redux/sagas/motions/expenditures/finalizeExpenditureMotion.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  ClientType,
+  Id,
+  getPermissionProofs,
+  ColonyRole,
+} from '@colony/colony-js';
+import { call, put, takeEvery } from 'redux-saga/effects';
+
+import { ADDRESS_ZERO, APP_URL } from '~constants/index.ts';
+import { type Action, ActionTypes } from '~redux/index.ts';
+import {
+  createGroupTransaction,
+  createTransactionChannels,
+  waitForTxResult,
+} from '~redux/sagas/transactions/index.ts';
+import {
+  getColonyManager,
+  initiateTransaction,
+  putError,
+  takeFrom,
+} from '~redux/sagas/utils/index.ts';
+
+export type FinalizeExpenditureMotionPayload =
+  Action<ActionTypes.MOTION_EXPENDITURE_FINALIZE>['payload'];
+
+function* finalizeExpenditureMotion({
+  meta,
+  meta: { setTxHash },
+  payload: { colony, votingReputationAddress, expenditure, motionDomainId },
+}: Action<ActionTypes.MOTION_EXPENDITURE_FINALIZE>) {
+  const { createMotion } = yield call(createTransactionChannels, meta.id, [
+    'createMotion',
+  ]);
+
+  try {
+    if (
+      !colony ||
+      !expenditure ||
+      !votingReputationAddress ||
+      !motionDomainId
+    ) {
+      throw new Error('Invalid payload');
+    }
+
+    if (colony.version < 15) {
```

Comment:
Can we get the colony version from colonyJS ? I know they all have tags, so would be nice if we could import it.

ie:
```
if (colony.version < ColonyVersion.GLWSS) {
```

Or something similar...

Reply:
we're still in development of v15 so that release has no tag yet I'm afraid

Reply:
Good point. In that case, ignore this :)